web/src/features/fbaCalculator/components/LoadingIndicatorCell.tsx

```diff
@@ -2,15 +2,16 @@ import { TableCell } from '@material-ui/core'
 import { Skeleton } from '@material-ui/lab'
 import React from 'react'
 
-interface LoadingIndicatorCellProps {
+export interface LoadingIndicatorCellProps {
   loading: boolean
   rowUpdating: boolean
   initialLoad: boolean
   children?: React.ReactNode
 }
 
 const LoadingIndicatorCell = (props: LoadingIndicatorCellProps) => {
-  const showLoadingIndicator = (props.loading && props.rowUpdating) || props.initialLoad
+  const showLoadingIndicator =
+    (props.loading && props.rowUpdating) || (props.initialLoad && props.loading)
```

Comment:
could be simplified to 
`props.loading && (props.rowUpdating || props.initialLoad)`
but now I'm just being pedantic